apps/api/src/app/groups/groups.service.ts

```diff
@@ -811,11 +811,26 @@ export class GroupsService {
      * Returns a list of groups.
      * @returns List of existing groups.
      */
-    async getGroups(filters?: { adminId: string }): Promise<Group[]> {
-        const where = []
+    async getGroups(filters?: {
+        adminId?: string
+        memberId?: string
+    }): Promise<Group[]> {
+        let where = {}
 
         if (filters?.adminId) {
-            where.push({ adminId: filters.adminId })
+            where = {
+                adminId: filters.adminId,
+                ...where
+            }
```

Comment:
```suggestion
            where = {
                adminId: filters.adminId
            }
```

The `where` is not necessary here, right?

Reply:
Right, I'll take it away.